bamboo-server/bamboo-server.sh

```diff
@@ -19,6 +19,14 @@ else
   echo "-> Extracting to $BAMBOO_DIR ..."
   tar xzf /tmp/atlassian-bamboo.tar.gz -C /opt
   rm -f /tmp/atlassian-bamboo.tar.gz
+  
+  echo "Add MySQL"
+  MYSQL_CONNECTOR_VERSION="5.1.38"
+  wget http://dev.mysql.com/get/Downloads/Connector-J/mysql-connector-java-$MYSQL_CONNECTOR_VERSION.tar.gz -O /tmp/mysql-connector.tar.gz
+  tar xzf /tmp/mysql-connector.tar.gz -C /tmp
```

Comment:
Extracting into /tmp is poor practice, I'd recommend creating a temporary directory (MYSQL_TMP_DIR=`mktemp -d`) and extracting into that. It has benefits, as shown in my next comment...